proposals/0000-markdown-content-redesign.md

```diff
@@ -0,0 +1,126 @@
+- Start Date: 03-10-2022
+- Reference Issues: https://github.com/withastro/rfcs/discussions/5, https://github.com/withastro/rfcs/discussions/118
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+A v2.0 API for working with local Markdown files in Astro.
+
+# Example
+
+```astro
+---
+// BASIC USAGE:
+
+// Note: Astro.fetchContent() removed in favor of direct `import.meta.globEager` usage
+// Note: This works in non-Astro framework components!
```

Comment:
This is a big plus, and honestly I could see this alone being a great reason to move away from `Astro.fetchContent()` 🎉